lib/modules/kss-splitter.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var gonzales = require('gonzales-pe'),
-    gonzo = require('gonzales-ast'),
+    gonzo = require('./node_traverser'),
```

Comment:
I think it wold be nice to change the name of this variable from `gonzo` to `traverser`. So it will be more clear that we are using our own module.